src/test/java/api/loans/OverdueFineScheduledNoticesProcessingTests.java

```diff
@@ -82,6 +84,7 @@ void uponAtNoticeIsSentAndDeleted(TriggeringEvent triggeringEvent) {
     assertThatScheduledNoticeExists(triggeringEvent, UPON_AT, false, actionDateTime);
 
     scheduledNoticeProcessingClient.runFeeFineNoticesProcessing(rightAfter(actionDateTime));
```

Comment:
Do we still need this processing to be triggered here?

Reply:
no, we don't